test/helper.go

```diff
@@ -0,0 +1,120 @@
+package test
```

Comment:
You'll probably want to rebase and make sure this still makes sense here after the e2e tests PR got merged. Also, i think you will need build tags on this

Reply:
moved files around. Removed duplicate helper function. Added build tags.